### PR TITLE
Clarifications to EIP-663

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

**Comment by axic** [comment]
@chfast @rodiazet please take a look

File: EIPS/eip-663.md [hunk 1]
```diff
@@ -36,14 +36,18 @@ If the code is legacy bytecode, both of these instructions result in an *excepti
 
 If the code is valid EOF1, the following execution rules apply:
 
-These instructions are followed by an 8-bit immediate value, which we call `imm`, and can have a value of 0 to 255. We introduce the variable `n` which equals to `imm + 1`.
+ 1. These instructions are followed by an 8-bit immediate value, which we call `imm`, and can have a value of 0 to 255. We introduce the variable `n` which equals to `imm + 17`.
 
-If the current stack depth is less than `n`, then a stack underflow exception is issued. If the current stack depth is at the limit, a stack overflow exception is issued. In both of these cases the EVM stops and all gas is consumed.
+ 2. For `DUPN`:
+   - If the current stack depth is less than `n`, then a stack underflow exception is issued.
```
Comment (chfast) [inline EIPS/eip-663.md:42]:
```suggestion
   - If the current stack height is less than `n`, then a stack underflow exception is issued.
```

File: EIPS/eip-663.md [hunk 2]
```diff
@@ -36,14 +36,18 @@ If the code is legacy bytecode, both of these instructions result in an *excepti
...
+   - If the current stack depth is at the limit (1024), a stack overflow exception is issued.
```
Comment (chfast) [inline EIPS/eip-663.md:43]:
```suggestion
   - If the current stack height is at the limit (1024), a stack overflow exception is issued.
```

File: EIPS/eip-663.md [hunk 3]
```diff
@@ -36,14 +36,18 @@ If the code is legacy bytecode, both of these instructions result in an *excepti
...
+   - Otherwise the stack item at depth `n` is duplicated at the top of the stack.
```
Comment (chfast) [inline EIPS/eip-663.md:44]:
```suggestion
   - Otherwise the `n`'th stack item is duplicated at the top of the stack.
```

File: EIPS/eip-663.md [hunk 4]
```diff
@@ -36,20 +36,24 @@ If the code is legacy bytecode, both of these instructions result in an *excepti
...
 
-Otherwise
+ 3. For `SWAPN`:
+   - If the current stack depth is less than `n + 1`, then a stack underflow exception is ussed.
```
Comment (chfast) [inline EIPS/eip-663.md:47]:
```suggestion
   - If the current stack height is less than `n + 1`, then a stack underflow exception is issued.
```

File: EIPS/eip-663.md [hunk 5]
```diff
@@ -36,20 +36,24 @@ If the code is legacy bytecode, both of these instructions result in an *excepti
...
+   - Otherwise the stack item at depth `n + 1` is swapped with the item at depth `1` (i.e. the top item).
```
Comment (chfast) [inline EIPS/eip-663.md:48]:
```suggestion
   - Otherwise the `n + 1`th stack item is swapped with the top stack item.
```

**Review by chfast — DISMISSED** [review]

**Pull request merged**